### ISSUE TITLE
The `set-output` command is deprecated 

### ISSUE THREAD

**Issue by Nimesha-Premaraja** [terraform-docs/gh-actions#109]
### Describe the bug

The `set-output` command is deprecated for GitHub Actions. We get following warning when run the GitHub Action

```
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

### How can we reproduce it?

should be updated to write to the new `GITHUB_STATE` and `GITHUB_OUTPUT` environment files as mentioned in this documentation
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

### Environment information

runs-on : ubuntu-latest
uses: terraform-docs/gh-actions@v1.0.0

**Comment by atrull** [comment]
+1

**Comment by actuarysailor** [comment]
+1

**Comment by ViacheslavKudinov** [comment]
Could be good to get this "concern" fixed.

**Comment by khos2ow** [comment]
Closing this in favor of its duplicate of #97 which is fixed in #94.
